src/constructors.jl

```diff
@@ -103,3 +103,24 @@ end
 function Base.call{T, N}(::Type{NullableArray{T, N}})
     NullableArray(T, ntuple(i->0, N))
 end
+
+#----- Conversion from Array of Nullables #1 -----------------------------------#
+# Take an Array of Nullable objects and convert it into a NullableArray
+function Base.convert{S,T,N}(::Type{NullableArray{S,N}},in::Array{Nullable{T},N})
```

Comment:
Thanks for the code review, @nalimilan. How will it work without `T`? For example, we don't have `Array{Nullable{Int,N}} <: Array{Nullable}`, and I didn't want this function to capture `Array`s that were filled with something non-`Nullable`.